css-brochure-form/styles.css

```diff
@@ -0,0 +1,67 @@
+h3 {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
+  text-align: center;
+  color: #ec5d57;
+  text-transform: uppercase;
+}
+
+form {
+  width: 50%;
+  margin: auto;
+}
+
+label {
+  margin-left: 15px;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```

Comment:
This line is not necessary here, it can be deleted.

Reply:
done.